style.css

```
@@ -0,0 +1,340 @@
+    -webkit-text-fill-color: transparent;
+    font-weight: 700;
+}
+#sec2 .left p{
+    color: #6B6B6B;
+    font-size: 1.6rem;
+    font-weight: 500;
+    line-height: 150%;
+}
+
+
+#sec3 .left h2{
+    font-size: 4.8rem;
+    font-weight: 700;
+}
+#sec3 .left h2 span{
+    background: linear-gradient(99deg, #6D7CCD 19.76%, rgba(82, 136, 133, 0.22) 52.69%);
+    background-clip: text;
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    /* font-size: 48em; */
+    font-weight: 700;
+}
+#sec3 .left p{
+    color: #6B6B6B;
+    font-size: 1.6rem;
+    font-weight: 500;
+    line-height: 150%;
+}
+
+
+#sec4 .left h2{
+    font-size: 4.8rem;
+    font-weight: 700;
+}
+#sec4 .left h2 span{
+    background: linear-gradient(99deg, #6D7CCD 19.76%, rgba(82, 136, 133, 0.22) 52.69%);
+    background-clip: text;
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    /* font-size: 48em; */
+    font-weight: 700;
+}
+#sec4 .left p{
+    color: #6B6B6B;
+    font-size: 1.6rem;
+    font-weight: 500;
+    line-height: 150%;
+}
+
+
+/* footer */
+footer{
+    background-color: #111322;
+    padding: 3.2rem 10.4rem 6.4rem;
+}
+footer .foot_inner{
+    max-width: 171.2rem;
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    color:#676767;
+    font-size: 1.6rem;
+}
+footer .foot_inner .mid{
+    color: #cfcfcf;
+    display: flex;
+}
+footer .foot_inner .mid a:first-child{
+    margin-right: 3.0rem;
+}
+footer .foot_inner .right a{
+    margin-right: 1.2rem;
+}
+footer .foot_inner .right a:last-child{
+    margin-right: 0;
+}
+
+/* signin,signup page */
+#signin{
+    background-color: var(--gray5);
+}
+.login{
+    width: 40.0rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%,-50%);
+    color: var(--black);
+}
+.login h1{
+    text-align: center;
+}
+.login h1 a img{
+    width: 21.0rem;
+    height: 3.8rem;
+    margin-bottom: 1.6rem;
+}
+.login p{
+    color: var(--black);
+    font-size: 1.6rem;
+    font-weight: 400;
+    text-align: center;
+}
+.login p a{
+    position: relative;
+    top: -0.5rem;
+    color: var(--blue);
+    text-decoration: underline;
+}
+
+.login form{
+    display: flex;
+    flex-direction: column;
+}
+.login .sign_top{
+    width: 100%;
+}
+.login .sign_top form .write{
+    margin: 3.0rem 0;
+    display: flex;
+    flex-direction: column;
+    gap: 1.2rem;
+}
+.login .sign_top form .write label{
+    font-size: 1.4rem;
+    font-weight: 400;
+}
+.login .sign_top form .write input{
+    width: 100%;
+    padding: 1.8rem 1.5rem;
+    border-radius: 0.8rem;
+    border: 0.1rem solid var(--gray3);
+}
+.login .sign_top form .write input:focus{
+    border: 0.1rem solid var(--blue);
+}
+.login .sign_top .pwd{
+    position: relative;
```

`relative`가 필요한 이유가 있으셨나요?!

---

그 안에있는 `.login .sign_top .pwd .eye`에
```suggestion
    position: absolute;
```
가 들어가 있어 주었습니다

---

아하..? 조금은 애매한데 좋은 대처네요!